packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -1,22 +1,6 @@
 @import '../../../../fxa-content-server/app/styles/breakpoints';
 
-.payment-error ~ .plan-details-component,
-.payment-confirmation + .plan-details-component {
-  margin: 16px 16px 32px;
-
-  @include max-width('tablet') {
-    grid-row: 3/4;
-  }
-}
-
-.plan-details-component {
-  @include min-width('tablet') {
-    height: 0px;
-  }
-
-
-  /* can be removed once '@tailwind base' is enabled */
-  .up-arrow::before {
-    transform: rotate(180deg);
-  }
+/* can be removed once '@tailwind base' is enabled */
+.up-arrow::before {
+  transform: rotate(180deg);
```

Comment:
Not sure if this matters, but previously this was wrapped in the following container:
```css
.plan-details-component {

}
```

But specificity or leakage might not matter since I _think_ we only have one instance of `.up-arrow` in the repo:
```sh
git grep -En "\bup-arrow\b" | cat

packages/fxa-payments-server/src/components/PlanDetails/index.scss:4:.up-arrow::before {
packages/fxa-payments-server/src/components/PlanDetails/index.tsx:379:   className="accordion-btn arrow before:rotate-180 up-arrow"
```

Reply:
Confirming that I checked that the only instance of `.up-arrow` was used in PlanDetails before I removed it from its wrapped `.plan-details-component`. :)